### DISCUSSION TITLE
2022.10.01 브리핑

### DISCUSSION THREAD

**Discussion by itmakesmesoft** [itmakesmesoft/Steady-Study#104] [Self-Assessment]
이번 주 목표를 이루기 위해 어떤 노력을 했는지 스스로 평가해보고, 점수를 매겨주세요.
ex)
이번 주 목표 : 기술 블로그에 배운 내용 정리하기

점수 : 80점
이유 : (생략)

### 월간 목표
이은혁 : 자바로 민코딩 시작반 1~2 풀기. 기술블로그 게시글 30개 채우기. 체중 3kg 감량(9월 30일 체중 인증)
고영석 : IM시험 대비하고, 합격하기!!
김지애 : 알고리즘 복습(swea or 백준 1일 1알고리즘 풀기), cs공부 시작하기
김지오 : IM 통과하기, 복싱+다이어트 식단, 데이터 공모전 1회 나가기, 깃 잔디 최대한 다 초록
나혜승 : IM시험 합격하기
이수연 : 정처기 필기 공부통해 CS 공부하기, 기술블로그 게시글 30개 채우기
이문삼 : 백준 30문제 풀기, 토이프로젝트 시작하기, 자바스크립트 강의 전부 듣기, 카카오 공채 지원해서 코테 체험하기(https://careers.kakao.com/2023-developer)

### 이번 주 목표 및 달성률(2022.09.25 ~ 2022.10.01)
이은혁 : 백준 1일 1문제씩 풀기, Velog 7개 글 올리기, 체중 62kg대 초중반 진입하기
고영석 : 월말평가 통과하기, 백준 IM 대비 문제 풀기, 블로그 3개 이상 작성하기
김지애 : 알고리즘 과제 전풀완, velog 개설하기+1개 작성하기, 운동 2회
김지오 : 백준 IM 대비 문제 매일 2개씩 풀기, 블로그 글 30개 넘게, 알고리즘 진도 따라가기 (현재를 살자)
나혜승 : 알고리즘 매일 2개씩 풀기, 문제 푼 것들 깃에 커밋하기, 알고리즘 수업 이해하기
이수연 : 블로그 글 7개 이상 쓰기, 깃허브 주 7회 올리기
이문삼 : 블로그 글 30개 넘기기, 자바스크립트 강의 듣기(제발)

**Comment by HyeseungNA** [comment]
목표 : 1일 알고 2문제, 문제들 깃에 커밋하기, 수업 이해하기
점수 : 80점
알고리즘 문제 매일 풀고 깃에 커밋했다. 수업내용도 관련 문제는 풀지 못했으나 이해를 하는데까지는 성공,,,

**Comment by YoungSeok222222** [comment]
이번달 목표: IM 대비하고 합격하기
이번주 목표: 월말평가 통과하기, 백준 IM 대비 문제 풀기, 블로그 3개 이상 작성하기


점수: 60점 
이유: 월말평가 (안 나옴), 백준 IM 대비 문제 풀기 (50%), 블로그 3개 이상 작성(완료) / 월말 목표는 IM 대비하고 합격하기지만, 10월달에 보기 때문에 대비는 하고 있지만, 아직 시험을 못봤기 때문에 다음달까지 보류할 예정입니다.

**Comment by moonthree** [comment]
주간 목표
목표 : 블로그 글 30개 넘기기, 자바스크립트 강의 듣기(제발)
점수 : 90점
이유 : 블로그 글 30개 넘김, 자바스크립트 강의 듣기 시작함

월간 목표
목표 : 백준 30문제 풀기, 토이프로젝트 시작하기, 자바스크립트 강의 전부 듣기, 카카오 공채 지원해서 코테 체험하기
점수 : 20점
이유 : 백준 30문제(실패), 토이프로젝트(실패), 자바스크립트(이제 시작), 카카오 공채(체험만 해봄)

**Comment by giokim12** [comment]
목표: 백준 IM 대비 문제 매일 2개씩 풀기, 블로그 글 30개 넘게, 알고리즘 진도 따라가기 (현재를 살자)
점수: 50점
IM대비 문제는 대체로 2개 이상씩 풀었고, 블로그 글도 30개 넘겼지만
현재에 사는거 그거 어떻게 하는건데... 

월간목표는 이룬게 없네유,,,

**Comment by sooyeonlee127** [comment]
목표 : 블로그 글 7개 이상 쓰기, 깃허브 주 7회 올리기
점수: 70점
블로그 글 9개 작성, 깃허브 커밋은 주 6회했습니다. 깃허브 커밋을 자꾸 까먹어서 기억하도록 노력하겠습니다....
월간 목표는 기술블로그 글 30개는 달성하였고, 정처기 필기는 2강밖에 못들었습니다 ㅠㅠ

**Comment by itmakesmesoft** [comment]
이번 주 목표 : 백준 1일 1문제씩 풀기, Velog 7개 글 올리기, 체중 62kg대 초중반 진입하기

점수 : 50점
이유 : 
- 백준 1일 1문제씩 풀기 - succeeded
- Velog 7개 글 올리기 - failed
- 체중 62kg대 초중반 진입하기 - abolutely failed

백준 1일 1문제 풀기 완료
벨로그 글은 4개 올리는데 그침
체중은 64kg 초과.....


이번 달 목표 : 자바로 민코딩 시작반 1~2 풀기. 기술블로그 게시글 30개 채우기. 체중 3kg 감량(9월 30일 체중 인증)
점수 : 30점
이유 : 
- 자바로 민코딩 시작반 1~2 풀기 - succeeded
- Velog 7개 글 올리기 - failed
- 체중 62kg대 초중반 진입하기 - abolutely failed

백준 1일 1문제 풀기 완료
벨로그 글은 4개 올리는데 그침
체중은 64kg 초과.....

**Comment by swjiae** [comment]
월간목표 : 알고리즘 복습(swea or 백준 1일 1알고리즘 풀기), cs공부 시작하기
주간목표 : 알고리즘 과제 전풀완, velog 개설하기+1개 작성하기, 운동 2회
점수 : 30점
알고리즘 전풀완은 전혀 못했지만 한문제 당 오래 고민해봤음
블로그 개설만함
운동 2회 성공
